app/code/Magento/CatalogImportExport/Test/Unit/Model/Import/Product/ValidatorTest.php

```diff
@@ -64,7 +65,10 @@ protected function setUp(): void
         $this->objectManagerHelper = new ObjectManagerHelper($this);
         $this->validator = $this->objectManagerHelper->getObject(
             Validator::class,
-            ['validators' => $this->validators]
+            [
+                'validators' => $this->validators,
+                'string' => new StringUtils()
+            ]
```

Comment:
Hello @SilinMykola 
Perhaps the parameters should have other positions to be compatible with all versions of PHP

```suggestion
            [
                'string' => new StringUtils(),
                'validators' => $this->validators
            ]
```

Reply:
Hello @andrewbess
Thanks for your review. I made changes, please check.